vitrina/datasets/templates/vitrina/datasets/coretable.html

```diff
@@ -178,5 +178,11 @@
             </tr>
         {% endif %}
     {% endif %}
+    <tr>
+        <th>{% translate "Duomenų rinkinio nariai" %}</th>
+        <td>
+            <a href="{% url 'dataset-members' org_kind=dataset.organization.kind org_slug=dataset.organization.slug dataset_slug=dataset.slug%}">{% translate "Peržiūrėti narių sąrašą"%}</a>
```

Comment:
```suggestion
            <a href="{{ dataset.get_members_url }}</a>
```

And dataset members should be accessed from dataset contextual menu, same way as it is done for orgs `vitrina/orgs/templates/vitrina/orgs/tabs.html`.